modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/EditAppSidebar.es.js

```diff
@@ -149,7 +162,7 @@ export default function EditAppSidebar() {
 					<div className="tab-title">
 						<ClayButton
 							displayType="secondary"
-							onClick={() => setCurrentTab(null)}
+							onClick={() => onClick(currentTab.onClickBack?.())}
```

Comment:
@gabrielpramos  At this case, you can just call the onClick,

Someone like this:

`onClick={onClick}`

And implement this:

`const onClick = () => {
        currentTab.onClickBack?.();
	setCurrentTab(null);
};`